modules/telariaBidAdapter.js

```diff
@@ -124,7 +124,7 @@ function getDefaultSrcPageUrl() {
 }
 
 function getEncodedValIfNotEmpty(val) {
-  return !utils.isEmpty(val) ? encodeURIComponent(val) : '';
+  return (val !== '') ? encodeURIComponent(val) : '';
```

Comment:
Hi @telariaEng , I assume that you're checking for `val !== ''` to avoid an unnecessary call to `encodeURIComponent`, because `encodeURIComponent('') === ''`.
What `val !== ''` lets through is `undefined`, so if some of the node properties are undefined, they will end up in your encoded schain string as string `undefined`. I'm just checking if this is intended behavior?

Reply:
Thank you for your response @aleksatr . I was actually avoiding not calling `encodeURIComponent(val)` for val = 0. Maybe I should just change it to `val !== undefined` ?